fiftyone/server/routes/sample.py

```
@@ -462,9 +487,24 @@ async def patch(self, request: Request, data: dict) -> JSONResponse:
-        )
+        response_body = utils.json.serialize(sample)
+
+        # Video sample responses must carry the first frame so the grid's
+        # video tile keeps its `sample.frames[0]` contract after a patch
+        # refresh propagates the new sample. `to_dict` drops `frames` by
+        # default; reconstruct just the head frame to match the grid's
+        # initial-load shape (a single-element list).
+        if sample.media_type == "video" and isinstance(response_body, dict):
+            try:
+                head_frame = sample.frames.first()
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)